### PR TITLE
Add WFMT as an ECS service

### PR DESCRIPTION
**Pull request by kookster**
- Adds a taskuser b/c the app needs AWS API keys
- Adds an SES user to be able to do email
- Uses shoryuken worker, so needs a queue for that
- Works with Porter, so that is configured
- Created the postgres dbs for stag and prod, add credentials to SSM
- Added in the SSM values for other secret values

### CODE REVIEW

**Review by farski — CHANGES_REQUESTED** [review]
Most of these changes are small typos or tag values.

The big thing to change, unless impossible, is the `TaskUser`.

File: stacks/apps/wfmt.yml [hunk 1]
```diff
@@ -0,0 +1,685 @@
...
+        - { Key: prx:cloudformation:root-stack-id, Value: !Ref RootStackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:ops:cloudwatch-log-group-name, Value: !Ref WorkerTaskLogGroup }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Wfmt }
+
+  TaskUser:
+    # TODO Someday when the AWS client within the Wfmt app has support for
+    # role-based authentication, these policies should be moved into the task
+    # role, and the user should be removed
```
[the remark this answers is not in the document]
Comment (kookster) [reply stacks/apps/wfmt.yml:252]: it is using an ancient fog-aws which only takes keys

[the remark this answers is not in the document]
Comment (farski) [reply stacks/apps/wfmt.yml:252]: Ah, ok

File: stacks/apps/wfmt.yml [hunk 2]
```diff
@@ -0,0 +1,685 @@
...
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:cloudformation:root-stack-name, Value: !Ref RootStackName }
+        - { Key: prx:cloudformation:root-stack-id, Value: !Ref RootStackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Wfmt }
+      TaskRoleArn: !GetAtt TaskRole.Arn
+Outputs:
+  TargetGroupFullName:
+    Value: !GetAtt WebTargetGroup.TargetGroupFullName
```
[the remark this answers is not in the document]
Comment (kookster) [reply stacks/apps/wfmt.yml:685]: ok, I see, will update

File: stacks/root.yml [hunk 3]
```diff
@@ -234,6 +235,7 @@ Resources:
       AdFilesHostname: a.prxu.org
       MetaHostname: meta.prx.org
       PublicFeedsHostname: !If [IsProduction, f.prxu.org, f-staging.prxu.org]
+      WfmtHostname: !If [IsProduction, wfmt.prx.org, wfmt.staging.prx.org]
```
[the remark this answers is not in the document]
Comment (kookster) [reply stacks/root.yml:238]: my bad, I know better

**Review by farski — APPROVED** [review]

**Pull request merged**